### PR TITLE
Fix image plot on lstm_conv example

### PR DESCRIPTION
**Pull request by jlsneto**
The error happens in the data visualization step

```python
# ...
ax.imshow(train_dataset[data_choice][idx], cmap='gray')
# ...
TypeError: Invalid shape (64, 64, 1) for image data
```

The fix is just to squeeze the image with `np.squeeze` function.

The fix has been applied to `examples/vision/ipynb/conv_lstm.ipynb`, `examples/vision/conv_lstm.py` and `examples/vision/md/conv_lstm.md`

### CODE REVIEW

File: examples/vision/conv_lstm.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def create_shifted_frames(data):
 # Plot each of the sequential images for one random data example.
 data_choice = np.random.choice(range(len(train_dataset)), size=1)[0]
 for idx, ax in enumerate(axes.flat):
-    ax.imshow(train_dataset[data_choice][idx], cmap="gray")
+    ax.imshow(np.squeeze(train_dataset[data_choice][idx]), cmap='gray')
```
Comment (fchollet) [inline examples/vision/conv_lstm.py:109]: Make sure to apply `black` formatting.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR. What version of matplotlib are you using?

**Comment by jlsneto** [comment]
> Thanks for the PR. What version of matplotlib are you using?

I'm using the default version installed on Google Collab, which is `3.2.2`

File: examples/vision/conv_lstm.py (hunk 1 above)
Comment (jlsneto) [reply examples/vision/conv_lstm.py]: Done!

**Review by fchollet — COMMENTED** [review]
Ok, sounds good! Please replicate the change in the ipynb and md files. The current ipynb file has many lines changed; prefer editing the original by hand since only a single line needs to change.

**Comment by jlsneto** [comment]
> Ok, sounds good! Please replicate the change in the ipynb and md files. The current ipynb file has many lines changed; prefer editing the original by hand since only a single line needs to change.

Thanks @fchollet!!!

**Comment by jlsneto** [comment]
I wasn't using the same version of black, but I've fixed it!

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Pull request merged**